src/app/containers/ATIAnalytics/params/media/index.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import * as atiPageViewParams from '../../atiUrl';
+import { ServiceContext } from '../../../../contexts/ServiceContext';
+import { RequestContextProvider } from '../../../../contexts/RequestContext';
+
+import MediaAtiParams from '.';
+import fixture from '../../../../../../data/indonesia/bbc_indonesian_radio/liveradio.json';
+
+describe('MediaAtiParams', () => {
+  const Component = (serviceContextStub, requestContextStub) => (
+    <ServiceContext.Provider value={serviceContextStub}>
+      <RequestContextProvider {...requestContextStub}>
+        <MediaAtiParams {...fixture} />
+      </RequestContextProvider>
+    </ServiceContext.Provider>
+  );
+  const requestContextStub = {
+    bbcOrigin: 'https://www.test.bbc.co.uk',
+    isAmp: false,
+    pageType: 'media',
+    service: 'SERVICE',
+  };
+  const serviceContextStub = {
+    atiAnalyticsAppName: 'news-SERVICE',
+    service: 'SERVICE',
+  };
+
+  describe('atiPageViewParams is called ', () => {
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('should call atiPageViewParams', () => {
+      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
+      atiPageViewParams.default = mock;
+
+      renderer.create(Component(serviceContextStub, requestContextStub));
```

Comment:
should we perhaps use `testing-library/react` library instead? I think there was just of exclusively using `testing-library/react` and eventually removing `react-test-renderer`